stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml

```diff
@@ -1,7 +1,6 @@
-{{- include "ibm-object-storage-plugin.licenseValidate" .  | required "License must be accepted by setting license key to true" }}
-
 ---
 # ServiceAccount for ibmcloud-object-storage-driver
+{{- include "ibm-object-storage-plugin.licenseValidate" .  | required "License must be accepted by setting license key to true" }}
```

Comment:
can you please revert this change, i.e. keep the line where it was - there is a known issue with GH page rendering if we move it after `---` line.

Reply:
@pedjak I do hit with the below error during chart install, if I move the line above `---`. Due to which I had to move it after `---`
```
Error: YAML parse error on ibm-object-storage-plugin/templates/flex-driver-sa.yaml: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal bool into Go value of type releaseutil.SimpleHead

Reply:
hm, we needed to for example here: https://github.com/redhat-developer/redhat-helm-charts/commit/2281025e8e4b8df93f0446fd14104c0350234f0d to introduce this week, otherwise GH pages are not regenerated...

Reply:
@pedjak I have reverted the above change and squashed the commits. Still, the PR scan seems to fail.

Reply:
there are still 2 commits in this PR, can you squash them into single one?

Reply:
Done.